pages/calling_conventions.adoc

```diff
@@ -8,18 +8,18 @@ description: Examples of calling conventions on common platforms.
 
 A *calling convention* is the set of contracts that compiler-generated machine
 code respects and expects external functions to respect.
-The calling convention specifies, for example
+The calling convention specifies various aspects, such as:
 
-- how parameters are passed to functions
+- how parameters are passed to functions.
 - how the stack is handled and cleaned (for example if needs to be aligned at
-  function entry)
-- how structures are going to be laid out in memory
-- which registers need to be restored by the caller and which by the callee
+  function entry).
+- how structures are going to be laid out in memory.
+- which registers need to be restored by the caller and which by the callee.
```

Comment:
should these be commans and `, and` on the penultimate item

Reply:
Seems like I made a mistake there. I checked the Wikipedia Manual of Style, and I think semicolons would be appropriate, though leaving it with no punctuation at all, would've also been okay (except the colon, which has to be there to introduce a list).